lib/src/okta/okta.guard.ts

```diff
@@ -100,22 +105,46 @@ export class OktaAuthGuard implements CanActivate, CanActivateChild, CanLoad {
     return this.canActivate(route, state);
   }
 
-  private async handleLogin(originalUri?: string): Promise<void> {
+  private async isAuthenticated(routeData?: Data, authState?: AuthState | null) {
```

Comment:
Should `canActivateChild` also have this check?

Reply:
`canActivateChild` just calls `callActivate`: https://github.com/okta/okta-angular/pull/132/files#diff-e5e5f0918a3176e60fa44ce0fe3d25a6311a8ed8f3c4a0113730eb3e0f5e941eR101-R106

Reply:
yep, gotcha, just had to actually look better. Thanks!